### PR TITLE
exa: add French translation

### PR DESCRIPTION
**Pull request by Ceres445**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/exa.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# exa
+
+> Un remplacement moderne pour `ls` (Lister le contenu du répertoire).
```
Comment (nicokosi) [inline pages.fr/common/exa.md:3]:
```suggestion
> Une alternative moderne à `ls` (pour lister le contenu de répertoires).
```

File: pages.fr/common/exa.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Liste des fichiers triés par date de modification (le plus ancien en premier) :
+
+`exa --long --sort={{modifié}}`
+
+- Répertorier les fichiers avec leur en-tête, leur icône et leur statut Git :
+
+`exa --long --header --icons --git`
+
+- Ne listez pas les fichiers mentionnés dans `.gitignore` :
```
Comment (nicokosi) [inline pages.fr/common/exa.md:34]:
```suggestion
- Lister les fichiers sauf ceux mentionnés dans `.gitignore` :
```

**Review by nicokosi — APPROVED** [review]
:ok: for me, even I propose suggestions. :relaxed:

File: pages.fr/common/exa.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# exa
+
+> Un remplacement moderne pour `ls` (Lister le contenu du répertoire).
+> Plus d'information: <https://the.exa.website>.
```
Comment (noraj) [inline pages.fr/common/exa.md:4]: > There must be a single blank space between special characters in the descriptions

see

https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#french-specific-rules

```suggestion
> Plus d'information : <https://the.exa.website>.
```

File: pages.fr/common/exa.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Lister les fichiers un par ligne :
```
Comment (noraj) [inline pages.fr/common/exa.md:6]: > Command and example descriptions on pages in French must use the third person singular present indicative tense

```suggestion
- Liste les fichiers, un par ligne :
```

File: pages.fr/common/exa.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Liste tous les fichiers, y compris les fichiers cachés :
+
+`exa --all`
+
+- Liste au format long (autorisations, propriété, taille et date de modification) de tous les fichiers :
+
+`exa --long --all`
+
+- Répertorier les fichiers avec le plus gros en haut :
```
Comment (noraj) [inline pages.fr/common/exa.md:18]:
```suggestion
- Liste les fichiers avec le plus volumineux en haut :
```

File: pages.fr/common/exa.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`exa --reverse --sort={{taille}}`
+
+- Afficher une arborescence de fichiers, sur trois niveaux de profondeur :
```
Comment (noraj) [inline pages.fr/common/exa.md:22]:
```suggestion
- Affiche une arborescence de fichiers, sur trois niveaux de profondeur :
```

File: pages.fr/common/exa.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`exa --long --tree --level={{3}}`
+
+- Liste des fichiers triés par date de modification (le plus ancien en premier) :
+
+`exa --long --sort={{modifié}}`
+
+- Répertorier les fichiers avec leur en-tête, leur icône et leur statut Git :
```
Comment (noraj) [inline pages.fr/common/exa.md:30]:
```suggestion
- Liste les fichiers avec leur en-tête, leur icône et leur statut Git :
```

File: pages.fr/common/exa.md (hunk 2 above)
Comment (noraj) [reply pages.fr/common/exa.md:34]:
```suggestion
- Liste les fichiers sauf ceux mentionnés dans `.gitignore` :
```

**Review by noraj — CHANGES_REQUESTED** [review]
Thanks for the contribution.

File: pages.fr/common/exa.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Liste des fichiers triés par date de modification (le plus ancien en premier) :
+
+`exa --long --sort={{modifié}}`
+
+- Liste les fichiers avec leur en-tête, leur icône et leur statut Git :
+
+`exa --long --header --icons --git`
+
+- Lister les fichiers sauf ceux mentionnés dans `.gitignore` :
```
Comment (noraj) [inline pages.fr/common/exa.md:34]:
```suggestion
- Liste les fichiers sauf ceux mentionnés dans `.gitignore` :
```

**Review by noraj — CHANGES_REQUESTED** [review]
The last one ;)

**Comment by Ceres445** [comment]
All done yay

**Review by noraj — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**